desk/app/groups.hoon

```diff
@@ -1456,6 +1456,11 @@
     ?>  =(p.flag our.bowl)
     =?  se-core  gone
       (se-update [%delete ~])
+    ::  if the group is about to be deleted, this our only
+    ::  chance to trigger +go-core update.
+    ::
+    =?  cor  gone
+      go-abet:(go-u-group:(go-abed:go-core flag) now.bowl %delete ~)
```

Comment:
Oh interesting, because the host _would_ sent out the `%delete` update, but the local `go` core would never handle it, because the group would've already been deleted from state here, so the update would no-op. Right?